test/notificationtest.cpp

```diff
@@ -285,8 +285,8 @@ void TestNotifications::testTextFilter() {
 void TestNotifications::testReverseClose() {
     const int timeout = 1000;
     const int max = 20;
-    static NotificationModel *m = new NotificationModel();
-    static NotificationServer *s = new NotificationServer(QDBusConnection::sessionBus(), *m);
+    NotificationModel *m = new NotificationModel();
+    NotificationServer *s = new NotificationServer(QDBusConnection::sessionBus(), *m);
```

Comment:
I think these changes to not use `static` here can be reverted. The variables occupy different space in memory, and since each test function only gets run once, as they aren't data driven, whether they are static or not shouldn't matter. Though, if we should remove the `static` usage, we should probably also pass `this` as the object parent, and remove the `delete` calls at the end of these two tests.